src/index.js

```diff
@@ -18,3 +18,4 @@ export { default as isSameDay } from './utils/isSameDay';
 export { default as toISODateString } from './utils/toISODateString';
 export { default as toLocalizedDateString } from './utils/toLocalizedDateString';
 export { default as toMomentObject } from './utils/toMomentObject';
+export { default as constants } from './constants';
```

Comment:
This isn't necessary; someone who wants them can require/import from `react-dates/lib/constants` already.

Reply:
Removed that line :)